version-stable-r-development.Rmd

```diff
@@ -212,22 +215,22 @@ If you are using LaTeX and start seeing errors like
 ```
 Remote repository is newer than local (2018 < 2019)
 ```
-it means that you have to re-install TinyTeX. This happens e.g. with
-`rocker/verse:3.6.1`, since it was build at the end of 2018 but the current
+you have to re-install TinyTeX. This happens e.g. with
+`rocker/verse:4.3.2`, since it was built at the end of 2018 but the current
 Tex Live repo is 2019. You can fix this via a **user-specific** re-installation of
 TinyTeX for R. **NOTE** however that this will uninstall the system-level
 TinyTeX pre-installed in `rocker/verse`.
 
 First, make sure `/home/rstudio/bin` is part of the `PATH` environment variable.
 Check this by running
 ```{bash check-path}
-docker exec --user rstudio rstudio_3.6.1 R --slave -e 'Sys.getenv("PATH")'
+docker exec --user rstudio rstudio_4.4.1 R --slave -e 'Sys.getenv("PATH")'
```

Comment:
when I tried to execute this, it failed with 
```
Error: unable to find user rstudio: no matching entries in passwd file
```
According to the [doc](https://github.com/rocker-org/rocker/wiki/Sharing-files-with-host-machine#avoiding-permission-changes-when-sharing-volumes) we linked re: permissions, it is only
> necessary to make sure that the UID of the user created (whether the it is the default user, rstudio or a custom username) matches the UID of the host user (the username is irrelevant, only the UID must match). 

When I go to the terminal tab in rstudio, the user is root (though the `$USERID` variable does match).
Will investigate, looks like there are some additional scripts in the newer rocker images.

Reply:
follow-up - it looks like the scripts I thought would be of interest are already executed when the standard `verse` image is built.

Reply:
Tried the concrete case again with 3.6.1, it seems versions <4 are not supported anymore (see the [rocker-versioned2 versions page](https://github.com/rocker-org/rocker-versioned2/wiki/Versions). I therefore would remove the section about fixing TinyTex for old versions.

Reply:
Stiil investigating - this is likely linked to an issue with podman which I'm using. I will add specific instructions once I've found out what the issue is.

Reply:
@spoltier, did you check also the currently-maintained documentation at https://rocker-project.org/, in particular https://rocker-project.org/use/rootless-podman.html ?

Reply:
Also https://rocker-project.org/use/shared_volumes.html

Reply:
In the end only had a minor issue with podman using the newer images from **rocker-versioned2**. See the added section.